src/client/utilities.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+
+  /**
+   * Converts an object of style {'a': {'b': c}, 'd': {'e': f}} to [{'b': c}, {'e': f}]
+   * @param {Object} givenObject Any object.
+   * @returns {Array}
+   */
+  'flattenObjectToArray': (givenObject) => {
+    let result = [];

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [`result` is never modified, use `const` instead.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2159922278/issues/source?bid=3148782&fileBranchId=3159641#l11)